app/design/frontend/Magento/blank/web/css/source/components/_modals_extend.less

```diff
@@ -152,6 +152,7 @@
 
 .media-width(@extremum, @break) when (@extremum = 'max') and (@break = @screen__m) {
     .modal-popup {
+        pointer-events: auto;
```

Comment:
remove pointer-events: none; from default media and change this one into "none" else it won't work.
Do the same with Luma theme.

Reply:
Hi @Karlasa ,

I have checked and found that in this pr there is current changes not present.